python/pyspark/sql/connect/streaming/readwriter.py

```diff
@@ -168,9 +168,75 @@ def json(
 
```

Comment:
Please ignore this change. They are added in https://github.com/apache/spark/pull/40689.